src/Modes/fault_index_type.enum

```diff
@@ -23,6 +23,7 @@ enum class fault_index_type : unsigned char
     sd_card,
```

Comment:
The reason your build is not passing is formatting- please run clang-format -i ./src/*.cpp ./src/*.hpp ./src/**/*.cpp ./src/**/*.hpp. See more here: https://github.com/Alpha-CubeSat/oop-flight-code/wiki/Coding-Standards